src/aws-serverless-mcp-server/README.md

```diff
@@ -78,8 +77,7 @@ Add the following code to your MCP client configuration. The Serverless MCP serv
           "AWS_REGION": "us-east-1"
         },
         "disabled": false,
-        "autoApprove": [],
-        "timeout": 60
```

Comment:
Nice I faced this same issue, setting it up. Can we get this out quickly? it degrades the UX without it.